connectors/src/connectors/slack/bot.ts

```diff
@@ -245,37 +262,55 @@ async function botAnswerMessage(
   if (mentions.length === 0) {
     mentions.push({ assistantId: "dust", assistantName: "dust" });
   }
-  // for now we support only one mention.
-  mentions = mentions.slice(0, 1);
-  const convRes = await dustAPI.createConversation({
-    title: null,
-    visibility: "unlisted",
-    message: {
-      content: message,
-      mentions: mentions.map((m) => {
-        return { configurationId: m.assistantId };
-      }),
-      context: {
-        timezone: slackChatBotMessage.slackTimezone || "Europe/Paris",
-        username: slackChatBotMessage.slackUserName,
-        fullName:
-          slackChatBotMessage.slackFullName ||
-          slackChatBotMessage.slackUserName,
-        email: slackChatBotMessage.slackEmail,
-        profilePictureUrl: slackChatBotMessage.slackAvatar || null,
-      },
+
+  const messageReqBody = {
```

Comment:
we pass all mentions now?

Reply:
We return an error if there is more than one mention.

Reply:
We should not we said we wanted to treat them as options not erroring?

Reply:
tbh I don't see the new error handling but I see a slice(0, 1) gone?

Reply:
We agreed on that [yesterday](https://dust4ai.slack.com/archives/C050SM8NSPK/p1696523341985719?thread_ts=1696517709.263099&cid=C050SM8NSPK).
The removed slice() was a left over from yesterday, it should not have been there.

The error handling for more than one mention is [here](https://github.com/dust-tt/dust/blob/main/connectors/src/connectors/slack/bot.ts#L198-L203).